src/js/video.js

```diff
@@ -62,7 +62,7 @@ var videojs = function(id, options, ready){
     }
 
     // If a player instance has already been created for this ID return it.
-    if (Player.players[id]) {
+    if (videojs.getPlayers()[id]) {
```

Comment:
`videojs` isn't imported in this file so this is relying on window.videojs. I think these should be switched back to use Player.